Emby.Server.Implementations/ConfigurationOptions.cs

```diff
@@ -1,13 +1,16 @@
 using System.Collections.Generic;
+using static MediaBrowser.Controller.Extensions.ConfigurationExtensions;
 
 namespace Emby.Server.Implementations
 {
     public static class ConfigurationOptions
     {
-        public static readonly Dictionary<string, string> Configuration = new Dictionary<string, string>
+        public static Dictionary<string, string> Configuration => new Dictionary<string, string>
         {
-            { "HttpListenerHost:DefaultRedirectPath", "web/index.html" },
-            { "MusicBrainz:BaseUrl", "https://www.musicbrainz.org" }
+            { "HttpListenerHost_DefaultRedirectPath", "web/index.html" },
+            { "MusicBrainz_BaseUrl", "https://www.musicbrainz.org" },
+            { FfmpegProbeSizeKey, "1G" },
```

Comment:
If there a reason why these two are constants and the other two are not?

Reply:
The orher 2 already existed, maybe I’ll do that in another PR